sdk/storage/Azure.Storage.Common/src/GeoRedundantReadResponseClassifier.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+
+using System;
+using System.Net;
+using Azure.Core.Pipeline;
+
+namespace Azure.Storage.Common
+{
+    internal class GeoRedundantReadResponseClassifier : ResponseClassifier
```

Comment:
Nit - add a comment to indicate it's used with RA-GRS/SecondaryUri for future readers

Reply:
Looks like I still need to add this.